src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -51,12 +51,13 @@ public without sharing class UTIL_OrgTelemetry_SVC {
      * against a table (such as a really huge Opportunity object) were to fail.
      */
     public enum TelemetryBatchCategory {
+        CustomInstallment_Settings,
         NPSP_Settings,
         Org_Environment,
         Data_MaxNumRelatedOpps,
         Data_CountRdOpps,
         Data_CountRecurringDonations,
-        Data_CountRds,
```

Comment:
This actually wasn't being used anywhere, I accidentally left this in, it was a duplicate for `Data_CountRecurringDonations`